.github/workflows/update-kernels.yml

```diff
@@ -21,13 +44,31 @@ jobs:
         with:
           ref: kernels
 
+      - name: Setup jq
```

Comment:
`jq` is already installed in ubuntu-latest image: https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Reply:
Oh we are actually running inside the kernel-crawler image :laughing: